controllers/buildpipeline/buildpipeline_adapter.go

```diff
@@ -64,6 +65,9 @@ func NewAdapter(pipelineRun *tektonv1.PipelineRun, component *applicationapiv1al
 // EnsureSnapshotExists is an operation that will ensure that a pipeline Snapshot associated
 // to the build PipelineRun being processed exists. Otherwise, it will create a new pipeline Snapshot.
 func (a *Adapter) EnsureSnapshotExists() (controller.OperationResult, error) {
+	var result string
```

Comment:
If I'm not missing something, I can't see where this value is set in the adapter. Could this variable (and the `status` parameter itself) be dropped and the `status` be set directly within the `annotateBuildPipelineRunWithCreateSnapshotAnnotation` function since that's where we figure out if something went wrong with Snapshot creation?

Reply:
Yep, done